analyzers/tests/SonarAnalyzer.UnitTest/Rules/AvoidExcessiveClassCouplingTest.cs

```diff
@@ -27,25 +27,27 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class AvoidExcessiveClassCouplingTest
     {
+        private readonly VerifierBuilder builderWithThreshold0 = new VerifierBuilder().AddAnalyzer(() => new AvoidExcessiveClassCoupling { Threshold = 0 });
+        private readonly VerifierBuilder builderWithThreshold1 = new VerifierBuilder().AddAnalyzer(() => new AvoidExcessiveClassCoupling { Threshold = 1 });
+
         [TestMethod]
         public void AvoidExcessiveClassCoupling() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\AvoidExcessiveClassCoupling.cs", new AvoidExcessiveClassCoupling { Threshold = 1 });
+            builderWithThreshold1.AddPaths("AvoidExcessiveClassCoupling.cs").Verify();
 
         [TestMethod]
         public void AvoidExcessiveClassCoupling_Generic_No_Constraints() =>
-            OldVerifier.VerifyCSharpAnalyzer(@"
+            builderWithThreshold0.AddSnippet(@"
```

Comment:
It's probably out of scope for this PR, but I find these snippets hard to read and maintain. If you share the feeling, please feel free to move them to a single new file inside TestCases directory.